app/javascript/packs/user_evaluations/evaluations.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Fetch from '../Fetch';
+
+
+class UserEvaluations extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      evaluations: [],
+    };
+    this.loadEvaluations = this.loadEvaluations.bind(this);
+  }
+
+  componentWillMount() {
+    this.loadEvaluations();
+  }
+
+  loadEvaluations() {
+    Fetch.json('/api/v1/evaluations')
+      .then((data) => {
+        this.setState({ evaluations: data });
+      }).catch(() => {
+        this.setState({ evaluations: false });
```

Comment:
we can have situations when the same object can have values of two types: array and boolean.